Source/vtkPyFR/vtkPyFRPipeline.cxx

```diff
@@ -645,31 +735,35 @@ int vtkPyFRPipeline::CoProcess(vtkCPDataDescription* dataDescription)
     this->InsituLink->InsituUpdate(dataDescription->GetTime(),
                                    dataDescription->GetTimeStep());
 
-    vtkUpdateFilter(this->Contour, dataDescription->GetTime());
-    vtkUpdateFilter(this->Slice, dataDescription->GetTime());
+      vtkUpdateFilter(this->Contour, dataDescription->GetTime());
+      vtkUpdateFilter(this->Slice, dataDescription->GetTime());
+
     if(this->PyData(dataDescription)->PrintMetadata()) {
-        double* bds = this->ContourMapper->GetBounds();
+        double* bds = this->ActiveMapper1->GetBounds();
         reduce(&bds[0], 1, vtkCommunicator::MIN_OP);
         reduce(&bds[1], 1, vtkCommunicator::MAX_OP);
         reduce(&bds[2], 1, vtkCommunicator::MIN_OP);
         reduce(&bds[3], 1, vtkCommunicator::MAX_OP);
         reduce(&bds[4], 1, vtkCommunicator::MIN_OP);
         reduce(&bds[5], 1, vtkCommunicator::MAX_OP);
         std::ostringstream b;
-        b << "[catalyst] world bounds: [ ";
+        //b << "[catalyst] (" << this->WhichPipeline << ") world bounds: [ ";
         std::copy(bds, bds+6, std::ostream_iterator<double>(b, ", "));
         b << "]\n";
```

Comment:
Doesn't really make sense to comment out *just* that line.
This is all gated on `PrintMetadata()`, which boils down to checking `metadata_out` in the .ini file.  So I think it's okay to be spammy; production runs should not have that set.